### PR TITLE
Rename beakerx_widgets to beakerx for BC

### PR DESCRIPTION
**Pull request by piorek**
https://github.com/twosigma/beakerx_base/pull/6

### CODE REVIEW

**Comment by EfimovVladimir** [comment]
I get the error while install beakerx widgets: "beakerx: command not found"
image

**Comment by piorek** [comment]
image

File: beakerx_widgets/beakerx/runtime.py [hunk 1]
```diff
@@ -135,7 +139,7 @@ def transformNaN(obj):
     if not isinstance(obj, float):
         return obj
     if math.isnan(obj):
-        return "Nan";
+        return "Nan"
```
Comment (afshin) [inline beakerx_widgets/beakerx/runtime.py:142]: typo `"Nan"` => `"NaN"`

Comment (piorek) [reply beakerx_widgets/beakerx/runtime.py]: fixed

**Comment by EfimovVladimir** [comment]
widgets doesn't works for another kernels (groovy, java, clojure, kotlin, scala)
image

**Pull request merged**